src/cli/config/index.ts

```diff
@@ -1,25 +1,27 @@
-const { resolve } = require('path');
-const { existsSync, readFileSync } = require('fs');
+import { resolve } from 'path';
+import * as fs from 'fs-extra';
 
-const cosmiconfig = require('cosmiconfig');
-const Joi = require('@hapi/joi');
-const { ValidationError } = require('@hapi/joi/lib/errors');
-const resolveFrom = require('resolve-from');
+import * as cosmiconfig from 'cosmiconfig';
+import * as Joi from '@hapi/joi';
+import { ValidationError } from '@hapi/joi/lib/errors';
+import * as resolveFrom from 'resolve-from';
+import * as JSON5 from 'json5';
```

Comment:
Some object destructuring?

Reply:
Always on the fence with this. Since it's just transpiring, and we are importing on load, destructuring has no performance gains (without tree shaking). Recommendations for these imports?

Reply:
Not all modules above have named import at the moment. This is good for now until we start looking into using `rollup`, `ncc` or `fusebox` for bundling.